examples/next/docs/js/demo/src/index.js

```diff
@@ -15,7 +15,9 @@ import {
 const example = document.getElementById("example");
 
 new Handsontable(example, {
-  data,
+  data: generateExampleData(),
+  layoutDirection: isArabicDemoEnabled() ? "rtl" : "ltr",
+  language: isArabicDemoEnabled() ? "ar-AR" : "en-US",
```

Comment:
Arabic language must be imported. See lines 8 and 33 here: https://github.com/handsontable/handsontable/blob/d8d13ff2ecf76279f9393c1617f59653b6de06a4/examples/next/docs/js/demo/src/index.js